### DISCUSSION TITLE
Add shortcut to node selection from 2d/3d editor

### DISCUSSION THREAD

**Discussion by PabloDevGit** [godotengine/godot-proposals#6549] [Editor]
### Describe the project you are working on

The issue has nothing to do with a project, the issue has to do with setting up shortcuts with my Logitech G604 mouse.

Problem:
The 2D editor does not have a shortcut to be able to move through the node tree from the 2D editor.
I can't configure a mouse button to move directly through the node tree because the 2D editor doesn't provide a shortcut for it.

### Describe the problem or limitation you are having in your project

Implement a shortcut for:
select top node
select bottom node
Currently there is only shortcut for MOVE node.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
Add a shortcut to be able to move through the node tree from the 2D editor.
...

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
A mouse button can be configured to assign that shortcut and move through the node tree through the mouse by pressing a single button.
...

### If this enhancement will not be used often, can it be worked around with a few lines of script?
You will press the key combination CTRL + numeric key + to select the node immediately above the current one or CTRL + numeric key - to select the node immediately below in the tree of nodes.
...

### Is there a reason why this should be core and not an add-on in the asset library?
Shortcuts are usually part of the core of any program.
...

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

PS: You should answer *every* question in the issue template form – these are not optional.
